recipes/simplecrop/build.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+mkdir src/_build
+cd src/_build
+
+cmake ../ -DCMAKE_INSTALL_PREFIX=$PREFIX
+make all install
```

Comment:
Is there a make check/test or ctest that we can run here?

Reply:
The only test I am aware of, unfortunately, is the smoke test I run in the tests section of *meta.yaml*, which just runs *simplecrop* on some example data files.